src/workerd/server/workerd.capnp

```diff
@@ -651,8 +663,17 @@ struct Worker {
   streamingTails @15 :List(ServiceDesignator);
   # List of streaming tail worker services that should receive tail events for this worker.
   # NOTE: This will be deleted in a future refactor, do not depend on this.
+
+  containerService @16 :ServiceDesignator;
```

Comment:
We don't need to make people configure `containerService` and `containerNetwork` like this. We can just take the `kj::Network` off the `AsyncIoProvider` and use it directly.

We should probably have some way for people to explicitly say that containers should be implemented using docker on the local machine, so we're not just automatically connecting to docker unexpectedly, and we can introduce other container engines later if desired. But this can simply be like:

```
containerEngine :union {
  none @16: void;
  localDocker @17: void;
}
```

(Use a union rather than an enum because some future engine may require more configuration.)